src/hotspot/share/prims/jvmtiAgent.hpp

```diff
@@ -43,6 +43,10 @@ class JvmtiAgent : public CHeapObj<mtServiceability> {
   const char* _options;
   void* _os_lib;
   const char* _os_lib_path;
+#ifdef AIX
+  unsigned long _inode;
+  unsigned long _device;
```

Comment:
It is best, IMO, to use the actual types rather than something expected to be "equivalent".

Reply:
OK, this would be ino64_t and dev64_t. I will do the change.